pkg/623398d0-43f3-4601-9cc4-cbce192a8521/package.lp

```diff
@@ -0,0 +1,81 @@
+(librepcb_package 623398d0-43f3-4601-9cc4-cbce192a8521
+ (name "PHOENIX_1985205")
+ (description "Spring-Cage PCB Termination Block with 3 positions, 45 degree mount with 3.5mm pitch")
+ (keywords "spring-cage")
+ (author "Raphael Nestler")
+ (version "0.1")
+ (created 2019-06-14T23:41:25Z)
+ (deprecated false)
+ (category 6183d171-e810-475a-a568-2a270aff8f5e)
```

Comment:
If we want to wait for https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/30/files:

```suggestion
 (category b724b55e-0f80-4cf3-aa9f-e2a87e02bd19)
```

(Alternatively we can put it in the pin socket category for now and change the category once the other PR is merged.)

Reply:
I think we leave it for now and may change it later, if https://github.com/LibrePCB-Libraries/LibrePCB_Base.lplib/pull/30 gets merged.